function main():
	a=$exception()
	a[123456]
+++exception INTEGER_OUT_OF_BOUNDS 0 3
